designdocs/privacy-design.md

```diff
@@ -0,0 +1,163 @@
+To accomplish this, mitigations (including but not limited to user consent) are outlined as either requirements or recommendations before allowing sites to access different types of data. An explanation of the mitigations and corresponding threat vectors can be found in the Background section.
+
+There is no distinction between different WebXR modes in this document. Data access requirements are the same whether the session is inline, immersive, or otherwise. If a user agent wants to avoid user consent for a particular type of session, the user agent should meet the requirements for all data available in that session. _For example, user agents that limit inline sessions to identity reference spaces only or apply necessary mitigations (such as data quantization) to eye-level and floor-level reference spaces may still offer them without first obtaining user consent._
+
+
+## Privacy Requirements
+### Conditions to create objects and expose data
+#### XRSession Creation
+
+The user agent must verify that all mandatory conditions are satisfied to ensure it can create an [XRSession](https://immersive-web.github.io/webxr/#xrsession-interface) and expose [XRSession](https://immersive-web.github.io/webxr/#xrsession-interface) data in response to a call to [XRRequestSession](https://immersive-web.github.io/webxr/#dom-xr-requestsession) within a given [active document](https://html.spec.whatwg.org/multipage/browsers.html#active-document).
+
+The **_mandatory conditions_** for [XRSession](https://immersive-web.github.io/webxr/#xrsession-interface) creation are the following:
+*   The document is a [responsible document](https://html.spec.whatwg.org/multipage/webappapis.html#responsible-document) of a [secure context](https://w3c.github.io/webappsec-secure-contexts/#secure-contexts).
+*   [Currently focused area](https://html.spec.whatwg.org/multipage/interaction.html#currently-focused-area-of-a-top-level-browsing-context) belongs to a document whose origin is [same origin-domain](https://html.spec.whatwg.org/multipage/origin.html#same-origin-domain) with the origin of the given [active document](https://html.spec.whatwg.org/multipage/browsers.html#active-document).
+*   The document is [allowed to use](https://wicg.github.io/feature-policy/#allowed-to-use) all the [policy-controlled features](https://wicg.github.io/feature-policy/#policy-controlled-feature) associated with the given [sensor types](https://www.w3.org/TR/2018/CR-generic-sensor-20180320/#sensor-type) of sensors used to generate data available in the [XRSession](https://immersive-web.github.io/webxr/#xrsession-interface). This includes sensors used to generate [XRViewerPose](https://immersive-web.github.io/webxr/#xrviewerpose), [XRPose](https://immersive-web.github.io/webxr/#xrpose-interface) and [XRBoundedReferenceSpace](https://immersive-web.github.io/webxr/#xrboundedreferencespace-interface) data, and any other data generated by sensors.
```

Comment:
Ah this is a case of GitHub being slightly too smart and formatting text in a way to hide the link. 

What I was asking is do we really want to be relying on the Devices and Sensors spec to be the exhaustive definition of data sources that can drive an XR system's tracking tech.  For example, the underlying accelerometer data might not actually be exposed as a Sensor on some devices.  Also cameras are a big part of XR tracking and is not exposed as a Sensor.

Reply:
We do not want the Devices and Sensors spec to be an exhaustive definition of data sources (nor is that the purpose of this section).

The purpose here is to avoid the scenario where developers use WebXR to work around the feature policies governing underlying sensor data. In the case of IMU data there is a fairly direct correlation between XRPose and IMU data, and it would be inappropriate to allow developers access to XRPose data (and thus IMU data) if feature policy prevented access to IMU data.

However, to your point the use of _camera_ data to generate XRPose falls into a different category, as it's unlikely that a developer could use XRPose to get camera data, and the camera isn't a sensor anyway.

I wonder if this section could be generalized in terms of respecting feature policies beyond sensors, but also more precise about the intent to prevent using WebXR as a 'back door' for feature policies.